src/main/java/edu/greenblitz/utils/GBMath.java

```diff
@@ -0,0 +1,17 @@
+package edu.greenblitz.utils;
+
+public class GBMath {
+	/**
```

Comment:
And again, the bare minimum is good docs.
The 'correct' implementation is not clear enough.
As I said many times, there are better names for this function (if it even needs to be used), but for the documentation to be good enough, it must include at least a numeric example or even just write that it is like python's module as it is the only popular programming language we saw that handles module like GB module.